src/citrine/resources/data_concepts.py

```diff
@@ -388,6 +388,102 @@ def register_all(self, models: List[ResourceType], dry_run=False) -> List[Resour
         )
         return [self.build(obj) for obj in response_data['objects']]
 
+    def async_update(self, model: ResourceType,
```

Comment:
Max 2 positional arguments.  Perhaps:
```suggestion
    def async_update(self, model: ResourceType, *, 
```